app/assets/stylesheets/application.scss

```diff
@@ -66,6 +73,24 @@
    //  @include vertical-align;
  }
 
+ .ghost-button-thick-border {
+   display: inline-block;
+   width: 200px;
+   padding: 8px;
+   font-weight: bold;
+   color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.